performance/scaling_data_ingestion.rst

```diff
@@ -72,12 +54,34 @@ Note that this query also takes an exclusive lock on the row in PostgreSQL, whic
 
 When the replication factor is 1, it is always safe to enable citus.all_modifications_commutative. Citus does not do this automatically yet.
 
+Bulk Copy (100-200k/s)
+----------------------
+
+Hash distributed tables support `COPY <http://www.postgresql.org/docs/9.5/static/sql-copy.html>`_ from the Citus master for bulk ingestion, which can achieve much higher ingestion rates than regular INSERT statements.
+
+COPY can be used to load data directly from an application using COPY .. FROM STDIN, or from a file on the server or program executed on the server.
+
+::
+
+    COPY counters FROM STDIN WTIH (FORMAT CSV);
+
+In psql, the \COPY command can be used to load data from the local machine. The \COPY command actually sends a COPY .. FROM STDIN command to the server before sending the local data, as would an application that loads data directly.
+
+::
+
+    psql -c "\\COPY counters FROM 'counters-20160304.csv' (FORMAT CSV)"
+
+
+A very powerful feature of COPY for hash distributed tables is that it asynchronously copies data to the workers over many parallel connections, one for each shard placement. This means that data can be ingested using multiple workers and multiple cores in parallel. Especially when there are expensive indexes such as a GIN, this can lead to major performance boosts over ingesting into a regular PostgreSQL table.
+
+To avoid opening too many connections to the workers. We recommend only running only one COPY command on a hash distributed table at a time. In practice, running more than two at a time rarely results in performance benefits. An exception is when all the data in the ingested file has a specific partition key value, which goes into a single shard. COPY will only open connections to shards when necessary.
```

Comment:
Should we highlight this as a "Note" if we deem it important? Will this result in shards being marked as inactive?